src/script/interpreter.cpp

```diff
@@ -1430,6 +1510,8 @@ void PrecomputedTransactionData::Init(const T& txTo, std::vector<CTxOut>&& spent
         m_spent_outputs_ready = true;
     }
 
+    // TODO: Improve this heuristic
+    bool uses_bip119_ctv = true;
```

Comment:
setting this to `false` still has `feature_checktemplateverify.py` passing

Reply:
This is to be expected, if uses_bip119_ctv is false, then the checker will just compute the values without the cache (unless taproot/143 flags passed -- i suppose we could tighten when the cache gets made). If it is true, then the cache is always available. The cache being present or not shouldn't break anything!